ckanext/unaids/custom_user_profile/actions.py

```diff
@@ -0,0 +1,157 @@
+import logging
+
+from flask_babel import force_locale
+
+import ckan.authz as authz
+from ckan import model, logic
+from ckan.common import _
+from ckan.lib.helpers import url_for, flash_success
+from ckan.lib.mailer import mail_user
+from ckan.plugins import toolkit
+from ckan.plugins.toolkit import config
+from ckanext.ytp_request.helper import get_safe_locale
+import ckanext.ytp_request.logic.action.create as ytp_request_action_create
+from ckanext.ytp_request.model import MemberRequest
+
+log = logging.getLogger(__name__)
+
+_SUBJECT_MEMBERSHIP_REQUEST = _("New membership request (%(organization)s)")
+_MESSAGE_MEMBERSHIP_REQUEST = _("""\
+    User %(user)s (%(email)s), %(job_title)s at %(affiliation)s, has requested membership to organization %(organization)s.
```

Comment:
Do we want this to be modern f-strings? Or is that a detail we are not so fussed about?

Reply:
You can't use fstrings here. Those are string templates with enabled translations. They're concretized later in `mail_new_membership_request` with the use of old `%` instead of the more modern `.format()`.
But as I commented earlier I wanted to keep the amount of changes to a minimum.